sdk/webpubsub/Azure.Messaging.WebPubSub.Client/src/Azure.Messaging.WebPubSub.Client.csproj

```diff
@@ -6,8 +6,7 @@
     <PackageTags>azure;webpubsub.client</PackageTags>
     <NoWarn>$(NoWarn);0067</NoWarn>
     <IncludeOperationsSharedSource>true</IncludeOperationsSharedSource>
-    <RequiredTargetFrameworks>netstandard2.1</RequiredTargetFrameworks>
-    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <TargetFrameworks>$(RequiredTargetFrameworks);netstandard2.1</TargetFrameworks>
```

Comment:
As a general rule, we'll want this package to have the same targets as the other Azure SDK packages - meaning that we should have just the `$(RequiredTargetFrameworks)` here.

If you have a strong reason to target `netstandard2.1` explicitly, we'll need to discuss with the architects and approval.  Can you help me understand the rationale for this, please?

Reply:
The consider here is if I target `netstandard2.1` explicitly, then I can use some `netstandard2.1` only interface/features. E.g. `Memory<byte>` overload in WebSocket client, which can improve the performance and reduce GCs. As it's a data plane SDK, we need consider the performance issue.

Reply:
I'm temporarily reverting the code for troubleshooting the ci failure issue.

Reply:
As mentioned, these are standardized across the Azure SDK packages and not something that we generally allow individual packages to deviate from.   We are having discussions about potentially adding `net6.0` in the near future, which would unlock those same multi-targeting scenarios.

If including `netstandard2.1` is something that you feel strongly about, it will require architect approval for an exception.  Please open a thread with @KrzysztofCwalina, @tg-msft, and myself to discuss.   Until that exception is approved, please keep this as `$(RequiredTargetFrameworks)` and remove any overrides.

Reply:
@jsquire I don't think there's a very strong justification now. I'll remove the overrides and looking forward adding net6.0